### PR TITLE
feat: Ops just recipes

### PR DESCRIPTION
**Pull request by Isaac-DeFrain**
## Describe your changes

Adds just recipes to improve dev experience around local deploys & testing.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have verified new and existing tests pass locally with my changes.
- [x] I verified whether it was necessary to increment the database version.

### CODE REVIEW

File: Justfile [hunk 1]
```diff
@@ -96,7 +100,7 @@ check:
 
 test-unit-mina-rs:
   @echo "--- Performing long-running mina-rs unit tests"
-  cd rust && time cargo nextest run --features mina_rs
+  cd rust && time cargo nextest run --release --features mina_rs
```
Comment (jhult) [inline Justfile:103]: Why??

**Review by jhult — APPROVED** [review]

File: Justfile (hunk 1 above)
Comment (Isaac-DeFrain) [reply Justfile:103]: because we do the release build for tier3 so we might as well use that binary for the unit tests

**Comment by n1tranquilla** [comment]
I will have to take some time to read through this more carefully and figure out what the are different improvements or problems being solved. Standby.

**Comment by Isaac-DeFrain** [comment]
FYI there are two major themes:
- ops: adds `shutdown` & `clean-*` just recipes
- logs: improves logs

File: ops/deploy.rb [hunk 2]
```diff
@@ -175,12 +200,17 @@
     success = false
   end
 
-  system(
-    "diff --unified #{IDXR_NORM_LEDGER} #{MINA_NORM_LEDGER}",
+  # check diff
+  unless system(
+    "diff --unified #{IDXR_NORM_LEDGER} #{MINA_NORM_LEDGER}".chomp,
```
Comment (jhult) [inline ops/deploy.rb:205]: `.chomp`??

Comment (Isaac-DeFrain) [reply ops/deploy.rb:205]: unnecessary...

Comment (Isaac-DeFrain) [reply ops/deploy.rb:205]: will remove

**Review by n1tranquilla — APPROVED** [review]

**Pull request merged**